### PR TITLE
[Transform] add regex for stand mode

### PR DESCRIPTION
**Pull request by jaeyun-jung**
add regex for stand mode, filter option string.

Signed-off-by: Jaeyun <jy1210.jung@samsung.com>

### CODE REVIEW

**Review by anyj0527 — APPROVED** [review]
LGTM 👍

File: gst/nnstreamer/tensor_transform/tensor_transform.c [hunk 1]
```diff
@@ -697,6 +693,13 @@ gst_tensor_transform_set_option_data (GstTensorTransform * filter)
       gchar **options = NULL;
       guint i, num_options;
 
+      if (!g_regex_match_simple (REGEX_STAND_OPTION, filter->option, 0, 0)) {
```
Comment (gichan-jang) [inline gst/nnstreamer/tensor_transform/tensor_transform.c:696]: How about setting `G_REGEX_CASELESS` as compile_options to ignore the case?

Comment (jaeyun-jung) [reply gst/nnstreamer/tensor_transform/tensor_transform.c]: updated

**Review by gichan-jang — APPROVED** [review]
LGTM

**Pull request merged**